cmd/utils.go

```diff
@@ -93,6 +93,22 @@ func path2BucketObject(s string) (bucket, prefix string) {
 	return path2BucketObjectWithBasePath("", s)
 }
 
+func getDefaultParityBlocks(drive int) int {
+	return drive / 2
+}
+
+func getDefaultDataBlocks(drive int) int {
+	return drive - getDefaultParityBlocks(drive)
+}
+
+func getReadQuorum(drive int) int {
+	return getDefaultDataBlocks(drive)
+}
+
+func getWriteQuorum(drive int) int {
+	return getDefaultDataBlocks(drive) + 1
```

Comment:
This is also too pessimistic, again with 13 drives it wants 8 drives for write quorum, whereas only 7 is needed.

```suggestion
	return (drive+2) / 2
```

Reply:
The reasoning here is that we atleast, write one parity extra regardless of how much data we have. This is to ensure that any data block missing we can safely reconstruct.